hawc/apps/udf/views.py

```diff
@@ -200,3 +201,17 @@ class DeleteTagBindingView(BaseDelete):
 
     def get_success_url(self):
         return self.assessment.get_udf_list_url()
+
+
+class UDFDetailMixin:
```

Comment:
I tried to add this mixin (instead of the changing the view's get_context_data directly) but it didn't seem to work. Any ideas?

Reply:
That should work fine, as long as the other classes call `super()`  Did you put the mixin on the far-left of the class inheritance?